src/app/search/search-filters.ts

```diff
@@ -992,6 +995,16 @@ function searchFilters(
         initDupes();
         return _lowerDupes[item.id];
       },
+      reacquirable(item: DimItem) {
+        if (
+          !flagEnum(item.collectibleState, DestinyCollectibleState.NotAcquired) &&
+          !flagEnum(item.collectibleState, DestinyCollectibleState.PurchaseDisabled)
```

Comment:
Personally I’d just directly use the bitwise and here and skip the flagEnum helper.